vip_spec.xsd

```diff
@@ -337,8 +340,11 @@ reference elements for polling location
         <xs:element name="Person">
           <xs:complexType>
             <xs:sequence>
+              <!-- SURVEY: Types xs:string and InternationalizedText seem
+                   to be used inconsistently here. Suggestions?
```

Comment:
Hrm, interesting. In that case I actually motion that was keep it as it is (i.e., the same as 1622.2) since the information will still be there and in the same format as 1622.2. We can add it to the list of suggestions for 1622.2 if there's a brief public comment period. Otherwise I can live with it as-is.